src/com/lftechnology/java/training/krishna/employeemanagement/dao/GenericDao.java

```diff
@@ -0,0 +1,65 @@
+
+package com.lftechnology.java.training.krishna.employeemanagement.dao;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * This program is used to perform different action such as create update.
+ *
+ * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+ * @version 1.0
+ */
+public interface GenericDao<T, PK extends Serializable> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z]$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00119)